### PR TITLE
VACMS-1041 Remove uninstalled modules.

### PR DESCRIPTION
**Pull request by swirtSJW**
## Description

See #1041

## QA steps

As user _uid_ with _user_role_
1. Do this
1. Then that
1. Then validate Acceptance Criteria from issue
- [ ] This
- [ ] That
- [ ] The other thing

## Definition of Done
- [ ] Product release notes 
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -11,13 +11,13 @@
...
     "require": {
         "cweagans/composer-patches": "^1.6.4",
-        "acquia/headless_lightning": "^1.4.0-beta2",
         "devshop/behat-drupal-extension": "dev-rewrite-last-page-output-links",
         "drupal-composer/drupal-scaffold": "^2.0.0",
         "drupal/address": "^1.4",
         "drupal/admin_toolbar": "^1.24",
         "drupal/auto_entitylabel": "^2.1@beta",
         "drupal/better_field_descriptions": "^1.4",
+        "drupal/blazy": "^2.0@RC",
```
Comment (swirtSJW) [inline composer.json]: This and all the additions below are modules that were not explicitly called out in our composer, but were brought in as dependencies of lightning and are currently enabled.

File: composer.lock [hunk 2]
```diff
@@ -3121,86 +2895,19 @@
                 "source": "https://git.drupalcode.org/project/better_field_descriptions"
             }
         },
-        {
-            "name": "drupal/bg_image_formatter",
```
Comment (swirtSJW) [inline composer.lock]: Not enabled, so letting it go.

File: composer.lock [hunk 3]
```diff
@@ -5314,81 +5017,30 @@
             }
         },
         {
-            "name": "drupal/entity_browser_block",
```
Comment (swirtSJW) [inline composer.lock]: Not enabled, so letting it go.

File: composer.lock [hunk 4]
```diff
@@ -6609,38 +6223,33 @@
             }
         },
         {
-            "name": "drupal/layout_builder_restrictions",
```
Comment (swirtSJW) [inline composer.lock]: Not enabled, letting it go.

File: composer.lock [hunk 5]
```diff
@@ -6650,1032 +6259,299 @@
...
+            "homepage": "http://drupal.org/project/libraries",
             "support": {
-                "source": "https://git.drupalcode.org/project/layout_builder_restrictions"
+                "source": "http://cgit.drupalcode.org/libraries",
+                "issues": "http://drupal.org/project/issues/libraries",
+                "irc": "irc://irc.freenode.org/drupal-contribute"
             }
         },
         {
-            "name": "drupal/layout_builder_st",
```
Comment (swirtSJW) [inline composer.lock]: not enabled. letting it go.

File: composer.lock [hunk 6]
```diff
@@ -6650,1032 +6259,299 @@
...
-                }
-            ],
-            "description": "Apply styles to blocks in Layout Builder.",
-            "homepage": "https://www.drupal.org/project/layout_builder_styles",
-            "support": {
-                "source": "https://git.drupalcode.org/project/layout_builder_styles"
-            }
-        },
-        {
-            "name": "drupal/layout_library",
```
Comment (swirtSJW) [inline composer.lock]: Not enabled. Letting it go.

File: composer.lock [hunk 7]
```diff
@@ -6650,1032 +6259,299 @@
...
-                }
-            ],
-            "description": "Provides a library of layouts for content-editors to select from",
-            "homepage": "https://www.drupal.org/project/layout_library",
-            "support": {
-                "source": "https://git.drupalcode.org/project/layout_library"
-            }
-        },
-        {
-            "name": "drupal/libraries",
```
Comment (swirtSJW) [inline composer.lock]: This was added back in higher up.

File: composer.lock [hunk 8]
```diff
@@ -9509,90 +8262,6 @@
                 "irc": "irc://irc.freenode.org/drupal-contribute"
             }
         },
-        {
-            "name": "drupal/panelizer",
```
Comment (swirtSJW) [inline composer.lock]: Not enabled.  Letting it go.

File: composer.lock [hunk 9]
```diff
@@ -9686,79 +8351,6 @@
                 "irc": "irc://irc.freenode.org/drupal-scotch"
             }
         },
-        {
-            "name": "drupal/panels_ipe",
```
Comment (swirtSJW) [inline composer.lock]: not enabled letting it go.

File: composer.lock [hunk 10]
```diff
@@ -10895,57 +9487,6 @@
                 "issues": "http://drupal.org/project/issues/seckit"
             }
         },
-        {
-            "name": "drupal/simple_gmap",
```
Comment (swirtSJW) [inline composer.lock]: Not enabled letting it go

**Comment by swirtSJW** [comment]
Looks like the slight update of envronment indicator creates a disply issue with the admin menu 
image

**Comment by swirtSJW** [comment]
It only seems to be an issue when this workflow menu is present AND the branchname is long 
image

**Comment by swirtSJW** [comment]
I solved the layout issue by reducing the length of the text.

**Comment by ethanteague** [comment]
> I solved the layout issue by reducing the length of the text.

Nice work, @swirtSJW

**Pull request merged**